hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestBlockReport2.java

```diff
@@ -89,13 +91,13 @@ private void waitForTempReplica(MiniDFSCluster cluster, Block bl, int DN_N1)
     final int TIMEOUT = 40000;
 
     if (LOG.isDebugEnabled()) {
-      LOG.debug("Wait for datanode " + DN_N1 + " to appear");
```

Comment:
is changing all the log.debug into log.info really needed?

Reply:
It is a unit test. It does not matter

Reply:
Is this test only for Hops? If yes it does not matter, if no changing lines of codes for no real purpose like that is a source of conflict when merging hadoop jiras.